sys/ps/ps.c

```diff
@@ -77,6 +78,7 @@ void ps(void)
             overall_stacksz += stacksz;
             stacksz -= thread_measure_stack_free(p->stack_start);
             overall_used += stacksz;
+            int isr_usage = thread_arch_isr_stack_usage();                         /* ISR stack usage */
```

Comment:
Yes, sorry - I missed that some lines before were collapsed in my view... never mind!